src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -85,7 +85,9 @@ protected override void OnDocumentClosing(DocumentId documentId)
 
         public virtual string GetFilePath(DocumentId documentId)
         {
-            return CurrentSolution.GetDocument(documentId)?.FilePath;
+            var solution = CurrentSolution;
+
+            return (solution.GetDocument(documentId) ?? solution.GetAdditionalDocument(documentId))?.FilePath;
```

Comment:
I think we should just introduce extension method 

Document GetAnyDocument(this Solution soluiton, DocumentId documentId) => solution.Getdocument(docuemntid) ?? solution.GetAdditionalDocument(documentId)

at the namespace where solution is defined.